commands.go

```diff
@@ -114,6 +117,31 @@ func (b *bot) sudoku(m dggchat.Message, s *dggchat.Session) {
 	s.SendMute(m.Sender.Nick, -1)
 }
 
+func (b *bot) frenchToastAlert(m dggchat.Message, s *dggchat.Session) {
+	type ftlXML struct {
+		XMLName xml.Name `xml:"frenchtoast"`
+		Status  string   `xml:"status"`
+	}
+	if !strings.HasPrefix(m.Message, "!frenchToastAlert") {
+		return
+	}
+	//get frenchToastAlert XML
+	resp, err := http.Get("https://www.universalhub.com/toast.xml")
+	if err != nil {
+		return
+	}
+	//Parse Alert
+	if resp.StatusCode != 200 {
+		b.sendMessageDedupe("Error reaching French Toast Alert", s)
+		return
+	}
+	byteValue, _ := ioutil.ReadAll(resp.Body)
+	var f ftlXML
+	xml.Unmarshal(byteValue, &f)
```

Comment:
Let's check the error to insure unmarshalling succeeded, if not then we would log out a message and return early.